src/app/privacy/page.tsx

```diff
@@ -0,0 +1,19 @@
+'use client';
+import React from 'react';
+import Link from "next/link";
+import { WrenchScrewdriverIcon } from '@heroicons/react/24/outline';
```

Comment:
I see the similar icon from carbon design system and we can use that for consistency purpose. 
https://carbondesignsystem.com/elements/icons/library/
You can search for `Tool Kit`

Reply:
Updated it.